### DISCUSSION TITLE
Attributes in with() is not decrypted

### DISCUSSION THREAD

**Discussion by chojisan** [spatie/laravel-ciphersweet#14] [Q&A]
The encrypted attributes on eagerload (with() and load()) is not decrypted.

Model::with('users:id,first_name,middle_name,lastname');

**Comment by freekmurze** [comment]
@chojisan Could you PR a failing test for this? If you already know the fix, please PR that as well.

**Reply by chojisan** [reply]
Sorry I don't have a failing test for this. Below  are the screenshot of my code and response. I'm using laravel 9 with jetstream.
![3](https://user-images.githubusercontent.com/7189861/182327943-91496b76-377f-4ed7-96f9-4587708e6ae6.JPG)
![1](https://user-images.githubusercontent.com/7189861/182328030-be3849f1-e44a-4f5c-9d22-d43e26470d8b.JPG)
![2](https://user-images.githubusercontent.com/7189861/182328060-f657d6ff-8033-433d-be4c-80dca28beb08.JPG)

I notice that if I include all the encrypted attributes, it decrypts it correctly but if only a selected attribute it does not decrypt.

**Comment by acewebsa** [comment]
This looks like a great project.
What I've found is that if I do the following:
`Table::all()`
or 
`Table::where('column', 'value')->get()`

The result will bring back all the rows with the value not encrypted.

If we use

`Table::whereBlind('column', 'column_index','value')->get(['encrypted_column'])`
or
`Table::all(['encrypted_column'])` 

Then the result will return the rows with an encrypted value.

So it only returns it encrypted if you specify the column specifically.
